From bd4a3186dc21f2c8d3e0f851cf262a34ddb6b625 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Fri, 4 Oct 2019 14:21:25 +0200
Subject: [PATCH] networkAgent: add support for SAE secrets

NetworkManager supports "WPA3 Personal" networks for some time now, they
use the SAE authentication. Add support for it alongside other
password-based mechanisms.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/751
---
 js/ui/components/networkAgent.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/js/ui/components/networkAgent.js b/js/ui/components/networkAgent.js
index 32d40fb2b..3ff957bf6 100644
--- a/js/ui/components/networkAgent.js
+++ b/js/ui/components/networkAgent.js
@@ -216,6 +216,7 @@ var NetworkSecretDialog = class extends ModalDialog.ModalDialog {
         // First the easy ones
         case 'wpa-none':
         case 'wpa-psk':
+        case 'sae':
             secrets.push({ label: _("Password: "), key: 'psk',
                            value: wirelessSecuritySetting.psk || '',
                            validate: this._validateWpaPsk, password: true });
-- 
2.32.0

